observability/mastra/src/exporters/json.ts

```
@@ -1140,18 +1511,18 @@ export class JsonExporter extends BaseExporter {
-    return [...this.#logs];
+    return [...this.#debugLogs];
   }
 
   /**
-   * Dump logs to console for debugging (uses console.error for visibility in test output)
+   * Dump debug logs to console for debugging (uses console.error for visibility in test output)
    */
   dumpLogs(): void {
     console.error('\n=== JsonExporter Logs ===');
-    this.#logs.forEach(log => console.error(log));
+    this.#debugLogs.forEach(log => console.error(log));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)